### PR TITLE
Dex trades beta dunesql

### PR DESCRIPTION
**Pull request by aalan3**
This is to test dex trades under the alias `trades_migration_beta` to make sure that everything builds correctly and to check the data.

### CODE REVIEW

File: models/dex/dex_trades_migration_beta.sql [hunk 1]
```diff
@@ -0,0 +1,133 @@
+{{ config(
+        tags=['dunesql'],
+        alias = alias('trades_migration_beta'),
+        partition_by = ['block_date'],
```
Comment (jeff-dude) [inline models/dex/dex_trades_migration_beta.sql:4]: `block_month` here to match pattern of upstream spells?

**Review by jeff-dude — APPROVED** [review]
good to go outside of partition question

**Comment by aalan3** [comment]
Good catch changed to `block_month` now.

**Review by antonio-mendes — APPROVED** [review]
Beautiful, looks good to me too

**Comment by aalan3** [comment]
This can't be merged until `block_time` exists on all models that are being referenced. I will create some PRs to fix this.

**Review by couralex6 — APPROVED** [review]

**Comment by aalan3** [comment]
I figured out the issue! The incremental part of statement was incorrectly formated due to the loop, so the last model didn't have the time filter (fixed in https://github.com/duneanalytics/spellbook/pull/4107/commits/b29627f1ddc32560733d47a527d22f1aea19c189) . This was not a problem before but since I added the `incremental_predicate`, the last model was inserting the entire model which caused duplicates because the destination has a time filter now.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**